freqtrade/freqai/prediction_models/CatboostRegressor.py

```diff
@@ -38,16 +38,13 @@ def fit(self, data_dictionary: Dict) -> Any:
                 weight=data_dictionary["test_weights"],
             )
 
+        init_model = self.get_init_model(dk.pair)
+
         model = CatBoostRegressor(
             allow_writing_files=False,
             **self.model_training_parameters,
         )
 
-        model.fit(X=train_data, eval_set=test_data)
-
-        # some evidence that catboost pools have memory leaks:
-        # https://github.com/catboost/catboost/issues/1835
-        del train_data, test_data
```

Comment:
we no longer fear from this memory leak?

Reply:
That leak was solved with my earlier mem-leak solution :-)